### PR TITLE
[W5.11][W13-1]Khoo Yuan Hui

### PR DESCRIPTION
**Pull request by alexiscatnip**
I made the find function case-insensitive and updated User Guide and Tests accordingly.

### CODE REVIEW

**Comment by bennchong** [comment]
Good and clear commits done after every minor changes to the code. Will help in the long run when finding bugs. Good job!

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 1]
```diff
@@ -25,8 +25,8 @@ public void execute() throws IllegalValueException {
         //same word, same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, different case: matched
```
Comment (okkhoy) [inline test/java/seedu/addressbook/commands/FindCommandTest.java:28]: Great! you updated the Junit Tests!

**Review by okkhoy — APPROVED** [review]
This enhancement surely gives you a tick mark on the graded LO.
However, not sure if the intended objective of being able to work with an intermediate size codebase was achieved.

**Comment by okkhoy** [comment]
@alexiscatnip 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**